miner/worker.go

```diff
@@ -798,6 +808,7 @@ LOOP:
 					inc:   true,
 				}
 			}
+			close(interruptCh)
```

Comment:
Prefer `defer close(interruptCh) ` after where we define the `interruptCh`

Reply:
maybe we want to set abort signal as early as possible ?

Reply:
The delay is ignorable, prefer code maintainability.

Reply:
updated